WooCommerce/WooCommerceTests/ViewModels/CardPresentPayments/PaymentCaptureOrchestratorTests.swift

```diff
@@ -0,0 +1,225 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+
+class PaymentCaptureOrchestratorTests: XCTestCase {
+
+    private var stores: MockStoresManager! = nil
+    private var sut: PaymentCaptureOrchestrator! = nil
+    private let sampleSiteID: Int64 = 1234
+
+    override func setUpWithError() throws {
+        stores = MockStoresManager(sessionManager: SessionManager.makeForTesting())
+        sut = PaymentCaptureOrchestrator(stores: stores,
+                                         paymentReceiptEmailParameterDeterminer: MockReceiptEmailParameterDeterminer())
+    }
```

Comment:
nit: I think we usually do setup/teardown using `override func setUp() { ... }`, if it doesn't throw an error?

in any case, I think calling `super` would be nice
```suggestion
    override func setUpWithError() throws {
        super.setUpWithError()
        stores = MockStoresManager(sessionManager: SessionManager.makeForTesting())
        sut = PaymentCaptureOrchestrator(stores: stores,
                                         paymentReceiptEmailParameterDeterminer: MockReceiptEmailParameterDeterminer())
    }
```